scripts/rollup/build.js

```diff
@@ -557,14 +562,15 @@ async function createBundle(bundle, bundleType) {
   }
 
   const importSideEffects = Modules.getImportSideEffects();
-  const pureExternalModules = Object.keys(importSideEffects).filter(
+  const moduleSideEffects = Object.keys(importSideEffects).filter(
     module => !importSideEffects[module]
   );
 
   const rollupConfig = {
     input: resolvedEntry,
     treeshake: {
-      pureExternalModules,
+      propertyReadSideEffects: false,
+      moduleSideEffects: id => !moduleSideEffects.includes(id),
```

Comment:
Personal opinion, the source code is best to ensure that the imported modules are no side effects, so we can set treeshake.moduleSideEffects to false, may be better, but I set false after the ci will not pass, now assign function type just to ci pass